proto/connectrpc/conformance/v1/config.proto

```diff
@@ -32,42 +32,61 @@ message Config {
   repeated ConfigCase exclude_cases = 3;
 }
 
+// Features define the feature set that a client or server support. They are
+// used to determine the tests that are possible to be run in a client-server
+// combination. They are defined in YAML files and are specified as part of the
+// --conf flag to the test runner.
```

Comment:
```suggestion
// Features define the feature set that a client or server supports. They are
// used to determine the server configurations and test casess that are 
// will be run. They are defined in YAML files and are specified as part of the
// --conf flag to the test runner.
```
Perhaps, though, much of this comment instead belongs on `Config`. After all `Config` is the schema from the `--conf` yaml file, not this message.